Source/WarpX.H

```diff
@@ -113,11 +114,27 @@ public:
 
     void Evolve (int numsteps = -1);
 
-    void EvolveImplicitPicardInit (int lev);
-    void SaveParticlesAtImplicitStepStart (WarpXParticleContainer& pc, int lev);
-    void FinishImplicitParticleUpdate (WarpXParticleContainer& pc, int lev);
-    void FinishImplicitFieldUpdate(amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > >& Efield_fp,
-                                   amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > >& Efield_n);
+    //
+    // Functions used by implicit solvers
+    //
+    void PreRHSOp ( amrex::Real  cur_time,
```

Comment:
Would it make sense to add `implicit` in the name of this function, so that other developers can more easy identify that this function relates to the implicit algorithm?
```suggestion
    void ImplicitPreRHSOp ( amrex::Real  cur_time,
```

Reply:
Agreed.